src/axom/export.h.in

```diff
@@ -0,0 +1,13 @@
+
+#ifndef AXOM_@_COMPONENT@_EXPORT_H
+#define AXOM_@_COMPONENT@_EXPORT_H
+
+#if defined(_MSC_VER)
+    #define AXOM_@_COMPONENT@_DEPRECATED __declspec(deprecated)
+#elif defined(__GNUC__) || defined(__clang__)
+    #define AXOM_@_COMPONENT@_DEPRECATED __attribute__((deprecated))
```

Comment:
Can't you use the deprecated attribute directly? It's in the c++14 standard (https://en.cppreference.com/w/cpp/language/attributes/deprecated).

You should be able to add the following to the checkInTriangle method and the deprecated message will give the alternative method to call.

`[[deprecated("Use contains() instead.")]]`

Reply:
Thanks -- I saw that, but thought it was a c++17 feature. (I didn't look too closely).

Question for the other reviewers: Should I revert the build system changes and just use `[[deprecated()]] instead`?

Reply:
Updated.